google/cloud/aiplatform/training_jobs.py

```diff
@@ -103,6 +103,65 @@ def __init__(
         self._project = project
         self._gca_resource = None
 
+    @classmethod
+    @abc.abstractmethod
+    def _supported_training_schemas(self) -> Tuple[str]:
```

Comment:
```suggestion
    def _supported_training_schemas(cls) -> Tuple[str]:
```

Reply:
The property decorator is appropriate here and is consistent with the abstract class property pattern we use in the SDK:

https://github.com/googleapis/python-aiplatform/blob/dev/google/cloud/aiplatform/base.py#L223

Concrete class usage:
https://github.com/googleapis/python-aiplatform/blob/dev/google/cloud/aiplatform/training_jobs.py#L72

Reply:
Ok I was wondering if they were compatible and took it off just to be safe as it didn't change too much.

I will add it back.